toffy/qc_comp.py

```diff
@@ -1,26 +1,21 @@
 import copy
-import matplotlib.pyplot as plt
 import os
+# needed to prevent UserWarning: low contrast image barf when saving images
+import warnings
+from shutil import rmtree
+
+import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
+import seaborn as sns
+from mibi_bin_tools import bin_files
 from requests.exceptions import HTTPError
 from scipy.ndimage import gaussian_filter
-import seaborn as sns
-from shutil import rmtree
+from tmi import image_utils, io_utils, load_utils, misc_utils
 
-from toffy.mibitracker_utils import MibiTrackerError
-from toffy.mibitracker_utils import MibiRequests
 from toffy import settings
-from tmi.image_utils import save_image
-
-import ark.utils.io_utils as io_utils
-import ark.utils.misc_utils as misc_utils
-import ark.utils.load_utils as load_utils
-from mibi_bin_tools import bin_files
+from toffy.mibitracker_utils import MibiRequests, MibiTrackerError
 
-
-# needed to prevent UserWarning: low contrast image barf when saving images
-import warnings
 warnings.filterwarnings('ignore', category=UserWarning)
```

Comment:
We don't need this anymore because of `save_image` right?

Reply:
Yeah, I'll remove it.